tasks/build-css.js

```diff
@@ -318,8 +319,6 @@ gulp.task('build-css:concat-standalone-md', function() {
     ])
       // .pipe(concat('spectrum-' + colorStop + '-md.css'))
       .pipe(concat('spectrum-' + colorStop + '.css'))
-      // Replace instances of & that refer to the colorstop selector with .secptrum
-      .pipe(replace(/^&/gm, '.spectrum'))
```

Comment:
This is no longer needed as the generated file already has `&` replaced by `.spectrum`.